code-reviews.md

```diff
@@ -28,6 +28,7 @@ accomplish this, everyone should follow these tips.
 - Ask questions rather than make statements. ("What do you think about...?")
 - Ask for clarification if the code or comments are unclear. ("I didn't understand.
   Can you clarify?")
+- Support opinions with reason and facts: "I recommend this approach instead because...."
```

Comment:
Hmm... to bikeshed the punctuation in the example statement or not to bikeshed :trollface: